test/ai.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { before, describe, it } from "mocha";
+import { OPENAI_ERROR, fetchUrlSummary } from "../bots/ai.js";
+import { expect } from "chai";
+
+describe("Fetching URL Summary", () => {
+  let openaiKey = process.env.OPENAI_API_KEY ?? "";
+
+  it("Failed to call without API KEY", async () => {
+    try {
+      await fetchUrlSummary("https://vercel.com", "");
+    } catch (err) {
+      expect(err.message).to.be.eql(OPENAI_ERROR);
+    }
+  });
+
+  it("Fetch the invalid url summary", async () => {
```

Comment:
I'm not sure what this is trying to test. I think if you add the success test case as described in the above comment, this test isn't needed.